com.chenyifaer.FaForever.yml

```diff
@@ -0,0 +1,82 @@
+id: com.chenyifaer.FaForever
+runtime: org.gnome.Platform
+runtime-version: '46'
+sdk: org.gnome.Sdk
+command: faforever
+
+finish-args:
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --share=network
+  - --device=dri
+  - --share=ipc
+  - --filesystem=xdg-download
+
+modules:
+  - name: webkit2gtk-4.0
```

Comment:
This should really be ported to use newer webkit2gtk. What needs 4.0?

Reply:
Reference this

https://github.com/flathub/in.cinny.Cinny/blob/6ff4f0273e4a5edcbb00cd7faca09e89071c0539/in.cinny.Cinny.yml#L32

Reply:
Is this tauri too?

Reply:
Yes